azext_iot/central/iot_central_api_preview/operations/_api_tokens_operations.py

```diff
@@ -0,0 +1,292 @@
+# coding=utf-8
```

Comment:
autorest generated SDKs should go into the `azext_iot/sdk/` directory, and under a central root makes sense `azext_iot/sdk/central/...`